client/app/pages/queries/query.html

```diff
@@ -107,6 +107,10 @@ <h3>
               <span ng-if="dataSource.options.doc_url == '' || !dataSource.options.doc_url">{{ dataSource.type_name }} documentation</span>
               <get-data-source-version id='data-source-version'></get-data-source-version>
 
+              <input type="checkbox" ng-model="editorAutocomplete">
+                Enable Autocomplete
+              </button>
```

Comment:
You probably want `<label><input type="checkbox"...>Enable Autocomplete</label>`